### PR TITLE
NC-629

### PR DESCRIPTION
**Pull request by root-beer**
:ballot_box_with_check:  Improve exception handling of Zabbix API wrapper

### CODE REVIEW

**Comment by root-beer** [comment]
Please review and merge PR. @Joeper214 :bow:

**Comment by Joeper214** [comment]
I'll review it now.

File: config/locales/en.yml [hunk 1]
```diff
@@ -531,6 +531,7 @@ en:
       start:          'Stopping Zabbix Server...'
       stop:           'Starting Zabbix Server...'
       not_running:    'Zabbix Server is not running.'
+      connrefused:    'Could not connect to Zabbix Server. Please check the configuration.'
```
Comment (Joeper214) [inline config/locales/en.yml]: :100:

**Comment by root-beer** [comment]
Sorry, I forgot updating Gemfile.lock.

**Comment by Joeper214** [comment]
Okay. Please update it first.

**Comment by root-beer** [comment]
I'm done!

**Comment by Joeper214** [comment]
:+1:

**Comment by Joeper214** [comment]
LGTM! Thanks. @root-beer  :bow:

**Comment by root-beer** [comment]
Thanks! :bow:

**Pull request merged**